### ISSUE TITLE
Improve "Where should we generate the template?" question

### ISSUE THREAD

**Issue by gr2m** [frontend-templates-world/new-snowpack-project#2]
The question was not clear to me at first. I think I would rephrase it to make clear it is about the path in which to generate the new app.

The question should accept `.` for the current folder as answer, and a default value could be derived from the selected template, or default to something like `my-snowpack-app`

**Comment by Olyno** [comment]
What could be better instead of ``Where should we generate the template?``?

**Comment by gr2m** [comment]
How about

> At what path should we generate the template?

**Comment by Olyno** [comment]
Done in 06d4054ee2019aba0fce37603ac7807e641d23c0 ✅
